articles/challenge_participation.md

```diff
@@ -37,48 +37,8 @@ The data stored on the challenge Synapse site can be accessed using the Synapse
 
 ## Run your Algorithms and Submit to the Challenge
 
-You can submit to a challenge queue by using the R, Python or web client. In many cases, submitting through the web is easiest. All submissions must be first uploaded into Synapse. Follow instructions [here](getting_started.md#project-and-data-management-on-synapse) on how to upload to a project. Most challenge queues will be labeled by `challengename-subchallenge#` as a challenge may have different questions that it may want you to answer.
+You can submit to a challenge queue by using the R, Python or web client. In many cases, submitting through the web is easiest. All submissions must be first uploaded into Synapse. Follow instructions [here](getting_started.md#project-and-data-management-on-synapse) on how to upload to a project. Most challenge queues will be labeled by `challengename-subchallenge#` as a challenge may have different questions that it may want you to answer.  Learn more about submitting to [evaluation queues](evaluation_queues.md)
```

Comment:
```suggestion
You can submit to a challenge queue by using the R, Python or web client. All submissions must be first uploaded to Synapse. Follow these [instructions](getting_started.md#project-and-data-management-on-synapse) to learn how to upload to a project. Most challenge queues will be labeled by `challengename-subchallenge#` as a challenge may have different questions that it may want you to answer.  Learn more about submitting to [evaluation queues](evaluation_queues.md)
```

Reply:
Most challenge are in fact structured this way.  For challenges that only ask one question, the queue name is usually

* challengename-submission

For those challenges that have multiple queues:

* challengename-subchallenge1
* challengename-subchallenge2

That being said, the text should probably just say, name your queue something that correlates with your challenge, because participants have to know which queue to submit to.